ckanext/qdes_schema/qdes_ckan_dataset.json

```diff
@@ -156,7 +157,7 @@
       "label": "Temporal coverage end",
       "display_property": "dcat:endDate",
       "preset": "date",
-      "validators": "qdes_temporal_start_end_date",
+      "validators": "qdes_temporal_start_end_date scheming_isodatetime",
```

Comment:
Same issue as `temporal_start`
Also when the field is saved in datetime format the validator `qdes_temporal_start_end_date` the next time because its expecting the value to be a string not datetime object

Reply:
refactored